tools/lib/provision.py

```diff
@@ -323,13 +335,21 @@ def install_yum_deps(deps_to_install: List[str]) -> None:
     # Later steps will ensure PostgreSQL is started
 
     # Link in tsearch data files
-    overwrite_symlink(
-        "/usr/share/myspell/en_US.dic",
-        f"/usr/pgsql-{POSTGRESQL_VERSION}/share/tsearch_data/en_us.dict",
+    run_as_root(
+        [
+            "ln",
+            "-nsf",
+            "/usr/share/myspell/en_US.dic",
+            f"/usr/pgsql-{POSTGRESQL_VERSION}/share/tsearch_data/en_us.dict",
+        ]
     )
-    overwrite_symlink(
-        "/usr/share/myspell/en_US.aff",
-        f"/usr/pgsql-{POSTGRESQL_VERSION}/share/tsearch_data/en_us.affix",
+    run_as_root(
+        [
+            "ln",
+            "-nsf",
+            "/usr/share/myspell/en_US.aff",
+            f"/usr/pgsql-{POSTGRESQL_VERSION}/share/tsearch_data/en_us.affix",
+        ]
```

Comment:
Why this change?  `overwrite_ symlink` should be equivalent.

Reply:
This requires root privileges. `overwrite_symlink` can't do it.

Reply:
OK, so this is a bug introduced in 5c20ee998c2c79a08fba8e2edc827fb058ab6c8d (basically because we haven't been testing the Fedora/RHEL logic at all since then).  

I think probably a good mini follow-up project would be to add the backend test suite running on Fedora 33 to our CI setup.